docs/configuring-playbook-bridge-mautrix-bridges.md

```diff
@@ -0,0 +1,203 @@
+
+**for all bridges with encryption support**:
+
+```yaml
+matrix_bridges_encryption_enabled: true
+matrix_bridges_encryption_default: true
+```
+
+**Alternatively**, for a specific bridge:
+
+```yaml
+matrix_mautrix_SERVICENAME_bridge_encryption_enabled: true
+matrix_mautrix_SERVICENAME_bridge_encryption_default: true
+```
+
+### Enable relay mode (optional)
+
+[Relay mode](https://docs.mau.fi/bridges/general/relay-mode.html) is off by default. Check [the table on the official documentation](https://docs.mau.fi/bridges/general/relay-mode.html#support-table) for bridges which support relay mode.
+
+If you would like to enable it, add the following configuration to your `inventory/host_vars/matrix.example.com/vars.yml` file:
+
+**for all bridges with relay mode support**:
+
+```yaml
+matrix_bridges_relay_enabled: true
+```
+
+**Alternatively**, for a specific bridge:
+
+```yaml
+matrix_mautrix_SERVICENAME_configuration_extension_yaml: |
+  bridge:
+    relay:
+      enabled: true
+```
+
+You can only have one `matrix_mautrix_SERVICENAME_configuration_extension_yaml` definition in `vars.yml` per bridge, so if you need multiple pieces of configuration there, just merge them like this:
+
+```yaml
+matrix_mautrix_SERVICENAME_configuration_extension_yaml: |
+  bridge:
+    permissions:
+      '@alice:{{ matrix_domain }}': admin
+    encryption:
+      allow: true
+      default: true
+```
+
+If you want to activate the relaybot in a room, send `!prefix set-relay` in the rooms where you want to use the bot (replace `!prefix` with the appropriate command prefix for the bridge, like `!signal` or `!wa`). To deactivate, send `!prefix unset-relay`.
+
+Use `!prefix set-pl 100` to be able for the bot to modify room settings and invite others.
+
+#### Allow anyone on the homeserver to become a relay user (optional)
+
+By default, only admins are allowed to set themselves as relay users. To allow anyone on your homeserver to set themselves as relay users, add the following configuration to your `inventory/host_vars/matrix.example.com/vars.yml` file:
+
+```yaml
+matrix_mautrix_SERVICENAME_bridge_relay_admin_only: false
+```
+
+### Set the bot's username (optional)
+
+To set the bot's username, add the following configuration to your `inventory/host_vars/matrix.example.com/vars.yml` file:
+
+```yaml
+matrix_mautrix_SERVICENAME_appservice_bot_username: "BOTNAME"
+```
+
+### Configure the logging level (optional)
+
+To specify the logging level, add the following configuration to your `inventory/host_vars/matrix.example.com/vars.yml` file:
+
+```yaml
+matrix_mautrix_SERVICENAME_logging_level: WARN
+```
+
+Replace WARN with one of the following to control the verbosity of the logs generated: TRACE, DEBUG, INFO, WARN, ERROR, or FATAL.
+
+If you have issues with a service, and are requesting support, the higher levels of logging will generally be more helpful.
```

Comment:
```suggestion
To specify the logging level, add the following configuration to your `inventory/host_vars/matrix.example.com/vars.yml` file:

```yaml
matrix_mautrix_SERVICENAME_logging_level: warn
```

Replace `warn` with one of the following to control the verbosity of the logs generated: `trace`, `debug`, `info`, `warn`, `error` or `fatal`.

If you have issues with a service, and are requesting support, the higher levels of logging (those that appear earlier in the list, like `trace`) will generally be more helpful.
```

bridgev2 bridges use [tulir/zeroconfig](https://github.com/tulir/zeroconfig) for logging.

Related to https://github.com/spantaleev/matrix-docker-ansible-deploy/commit/f8b779f8ed36426eeb3db2e27aa6826d7e65f506

Reply:
Addressed with https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/3913/commits/40f34e541b9a6887b7ee07ccad27eb77210133da